custom_components/jcihitachi_tw/sensor.py

```diff
@@ -107,6 +107,8 @@ def native_value(self):
         """Return the PM2.5 value."""
         status = self.hass.data[DOMAIN][UPDATED_DATA].get(self._thing.name, None)
         if status:
+            if status.pm25_value == "unsupported":
+                return None
             return status.pm25_value
```

Comment:
Please use one-line comprehension:
```suggestion
            return None if status.pm25_value == "unsupported" else status.pm25_value
```

Reply:
Fixed